include/circt/Dialect/FIRRTL/FIRParser.h

```diff
@@ -15,6 +15,8 @@
 #define CIRCT_DIALECT_FIRRTL_FIRPARSER_H
 
 #include "circt/Support/LLVM.h"
+#include <string>
+#include <vector>
```

Comment:
One alternative to this is to forward declare these as opposed to an actual include. That may work here.